scapy/layers/http.py

```diff
@@ -310,8 +303,15 @@ def post_dissect(self, s):
             elif "compress" in encodings:
                 import lzw
                 s = lzw.decompress(s)
-            elif "br" in encodings and is_brotli_available:
-                s = brotli.decompress(s)
+            elif "br" in encodings:
+                try:
+                    import brotli
```

Comment:
I will suggest to set the is_brotli_available variable once if the import is ok or not to do the import only once. It is a good idea to move the error message from the top of the file.